dotnetapp-dev/README.md

```diff
@@ -26,6 +30,60 @@ docker run --rm dotnetapp-dev Hello .NET Core from Docker
 
 Note: The instructions above work for both Linux and Windows containers. The .NET Core docker images use [multi-arch tags](https://github.com/dotnet/announcements/issues/14), which abstract away different operating system choices for most use-cases.
 
+## Run unit tests as part of `docker build`
+
+The unit tests will have run as part of the the `docker build` command listed above. You can make the unit test fail by changing the [unit test](tests/UnitTest1.cs) to match the test below.
+
+```csharp
+[Fact]
+public void Test1()
+{
+    var inputString = "Dotnet-bot: Welcome to using .NET Core!";
+    var expectedString = "Dotnet-bot: Welcome to using .NET Core!";
+    var actualString = ReverseUtil.ReverseString(inputString);
+    Assert.True(actualString == expectedString, "The input string was not reversed correctly.");
+}
+```
+
+After changing the test, re-run `docker build` so that you can see the failure, with the following command.
+
+```console
+docker build -t dotnetapp-dev .
+```
+
+## Run unit tests as part of `docker run`
+
+The sample runs unit tests as part of `docker build`, as described above. That's useful as a means of getting feedback during `build` (the build will fail), but there isn't an easy way to get the test logs. The sample exposes a test stage that you can build and then run explicity. You can cause unit tests logs to be written to your local disk for viewing outside of a container by using [volume mounting](https://docs.docker.com/engine/admin/volumes/volumes/).
```

Comment:
What do you think about saying `The samples exposes a test runner stage` instead of `test stage`?